internal/cmd/restore.go

```diff
@@ -22,6 +22,7 @@ func registerRestoreCmd(rootCmd *cobra.Command) {
 	rootCmd.AddCommand(restoreCmd)
 	restoreCmd.Flags().Int("batch-size", 1_000, "restore relationship write batch size")
 	restoreCmd.Flags().Int("batches-per-transaction", 10, "number of batches per transaction")
+	restoreCmd.Flags().Bool("print-zedtoken-only", false, "just print the zedtoken and stop")
```

Comment:
Maybe this should be another command entirely?

Reply:
I mean, I considered it, and thought that this would be more discoverable.